package.json

```diff
@@ -3,10 +3,10 @@
   "version": "1.0.0",
   "main": "index.js",
   "scripts": {
+    "build": "tsc",
```

Comment:
build added for compiling tsc to js on the dist folder

Reply:
Ok nice